python/milvus_storage/_ffi.py

```diff
@@ -0,0 +1,269 @@
+"""
+Low-level FFI bindings for milvus-storage C API.
+
+This module provides cffi wrappers around the C API defined in ffi_c.h.
+"""
+
+import os
+import platform
+from typing import Optional
+from cffi import FFI
+
+
+# Error codes from ffi_c.h
+LOON_SUCCESS = 0
+LOON_INVALID_ARGS = 1
+LOON_MEMORY_ERROR = 2
+LOON_ARROW_ERROR = 3
+LOON_LOGICAL_ERROR = 4
+LOON_GOT_EXCEPTION = 5
+LOON_UNREACHABLE_ERROR = 6
+LOON_INVALID_PROPERTIES = 7
+
+
+# Create FFI instance and define C API
+_ffi = FFI()
+
+# Define C structures and function signatures
+_ffi.cdef("""
```

Comment:
replace with 
```
with open("ffi_c.h") as f:
    ffi.cdef(f.read())

with open("abi.h") as f:
    ffi.cdef(f.read())
``` 
?

or after `ffi_c.h` updated, we also need update this file.

Reply:
replace with
> 
> ```python
> with open("ffi_c.h") as f:
>     ffi.cdef(f.read())
>
> with open("abi.h") as f:
>     ffi.cdef(f.read())
```
 
?
```
> Or after `ffi_c.h` updated, we also need to update this file.

no, as some C++ syntaxes are not supported in cdef.